tutorials/tts/Fastpitch_Training_GermanTTS.ipynb

```diff
@@ -0,0 +1,1112 @@
+    "wavs\n",
+    "\n",
+    "/Data/thorsten-de/wavs:\n",
+    "00025a6fbea659dae6ece011e749aa34.wav\n",
+    "000314280388fb390b3e70b69ee53a23.wav\n",
+    "000624f768d7e282534a850980619fb2.wav\n",
+    "...\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f8dee84",
+   "metadata": {},
+   "source": [
+    "## 3. Augmenting dataset with phonemes\n",
+    "\n",
+    "The original dataset only contains text input, however we want our model to be able to accept both text and phonemes. So, we need to convert our text into phonemes. We will be using [bootphon/phonemizer](https://github.com/bootphon/phonemizer) from GitHub to convert German text to phonemes. Example: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4f3aaa1a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"text: Geht die Schandtat auf sein Konto?\")\n",
+    "print(\"phoneme: \\u0261e\\u02d0t di\\u02d0 \\u0283ant\\u0251\\u02d0t a\\u028af za\\u026an k\\u0254nto\\u02d0\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b468c138",
+   "metadata": {},
+   "source": [
+    "\n",
+    "The quickest way to get started with phonemizer is to use docker container:\n",
+    "```bash\n",
+    "git clone https://github.com/bootphon/phonemizer\n",
+    "cd phonemizer\n",
+    "docker build -t phonemizer .\n",
+    "docker run --rm -d -it -p 8888:8888 -v /Data:/Data --ipc=host phonemizer /bin/bash\n",
+    "docker exec -it <docker_container_id> /bin/bash\n",
+    "pip install jupyterlab\n",
+    "jupyter lab --ip=0.0.0.0 --allow-root --NotebookApp.token='' --notebook-dir=/ --NotebookApp.allow_origin='*' &\n",
+    "```\n",
+    "\n",
+    "Within the container, run the following python script:\n",
+    "\n",
+    "```python\n",
+    "from phonemizer.backend import EspeakBackend\n",
+    "import json\n",
+    "\n",
+    "backend = EspeakBackend('de')\n",
+    "\n",
+    "input_manifest_filepath = \"/Data/thorsten-de/train_manifest.json\"\n",
+    "output_manifest_filepath = \"/Data/thorsten-de/train_phonemes_manifest.json\"\n",
+    "\n",
+    "records = []\n",
+    "n_text = []\n",
+    "with open(input_manifest_filepath, \"r\") as f:\n",
+    "    for i, line in enumerate(f):\n",
+    "        d = json.loads(line)\n",
+    "        records.append(d)\n",
+    "        n_text.append(d['normalized_text'])\n",
+    "\n",
+    "phonemized = backend.phonemize(n_text)\n",
+    "\n",
+    "new_records = []\n",
+    "for i in range(len(records)):\n",
+    "    records[i][\"phoneme\"] = 0\n",
+    "    new_records.append(records[i])\n",
+    "    phoneme_record = records[i].copy()\n",
+    "    phoneme_record[\"normalized_text\"] = phonemized[i]\n",
+    "    phoneme_record[\"phoneme\"] = 1\n",
+    "    new_records.append(phoneme_record)\n",
+    "    \n",
+    "with open(output_manifest_filepath, \"w\") as f:\n",
+    "    for r in new_records:\n",
+    "        f.write(json.dumps(r) + '\\n')\n",
```

Comment:
Is there a way to make this runnable from within the notebook (i.e. without relying on starting a container) such that people on Colab can try it?